.github/workflows/node.js.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x]
```

Comment:
That's a broad version range. Do we need that, or can this project pick one version to build on?

Reply:
this is just for publishing the site so picking one that works the best would be good. something not too old or too new I suppose :?

Reply:
14 is the current LTS branch, so let's pick that? https://nodejs.org/en/about/releases/

Reply:
Ya , I think 14 would be enought for our purpose.